src/CodeGeneration/ApiGenerator/RestSpecification/Core/bulk.json

```diff
@@ -1,6 +1,6 @@
 {
   "bulk": {
-    "documentation": "http://www.elastic.co/guide/en/elasticsearch/reference/2.3/docs-bulk.html",
+    "documentation": "http://www.elastic.co/guide/en/elasticsearch/reference/master/docs-bulk.html",
```

Comment:
I updated the rest api specs to point at `v.2.4.0` to pull in the spec for reindex rethrottle. [The urls have not been updated on that branch](https://github.com/elastic/elasticsearch/blob/v2.4.0/rest-api-spec/src/main/resources/rest-api-spec/api/bulk.json#L3).